README.md

```diff
@@ -108,7 +108,10 @@ as the communication channel for both feature requests and issues.
 
 This project does not accept *Public* contributions at this time.
 
-**Microsoft-authorized contributors - please refer to contributing instructions [here](https://github.com/microsoft/cpp_client_telemetry_modules/blob/master/CONTRIBUTING.md).**
+**[Microsoft-authorized
```

Comment:
The line-break here looks weird, is that intentional?

Reply:
Yes, it's automated using `rewrap` and I confirm this is the intended behavior.